spec/lib/mina/runner/pretty_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+describe Mina::Runner::Pretty do
+  subject(:runner) { described_class.new('true') }
+
+  describe '#run', :suppressed_output do
+    it 'executes the script' do
+      expect { runner.run }.to output('').to_stdout
```

Comment:
https://github.com/mina-deploy/mina/pull/686#pullrequestreview-730075794

@d4be4st this spec actually executes the command (in this case `true`, which outputs nothing). So that at least covers calling `Open3.popen3` and waiting for the process to end.

The part that is not tested (and I'm not sure how to test it) is handling SIGINT and output formatting.